tests/dummy/app/templates/public-pages/docs/contributing.hbs

```diff
@@ -0,0 +1,29 @@
+<h1 class="doc-page-title">Contributing</h1>
+
+<p>
+  If you would like to contribute, you can fork the project, edit, and make a
+  pull request. Make sure to write tests.
+</p>
+
+<h3>Update the website & documentation</h3>
+
+<p>
+  The documentation website lives under <code>tests/dummy</code>. It is a simple
+  ember app. For instance you can find this page in the
+  <code>tests/dummy/public-pages/docs/contributing.hbs</code> template. You
+  can run the website locally:
+</p>
+
+<pre>
+  <code>
+    npm install
+    bower install
+    cd tests/dummy
```

Comment:
Check the other comment. I think this explanations fit better in the readme of the project than in the web.